ui-tests/src/test/java/org/openmrs/reference/BookRequestAppointmentTest.java

```diff
@@ -42,8 +42,8 @@ public void bookRequestAppointmentTest() throws Exception {
         ClinicianFacingPatientDashboardPage patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         RequestAppointmentPage requestAppointmentPage = patientDashboardPage.clickOnRequest();
         requestAppointmentPage.enterAppointmentType("Oncology");
-        requestAppointmentPage.enterValue("0");
-        requestAppointmentPage.selectUnits("Day(s)");
+        requestAppointmentPage.enterMinimumValue("0");
+        requestAppointmentPage.selectMinimumUnits("Day(s)");
```

Comment:
Refactored the method names [h](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/aa118a51a46e504e7b3e534180ca74bdd36bc546/ui-tests/src/test/java/org/openmrs/reference/page/RequestAppointmentPage.java#L31)er[e](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/aa118a51a46e504e7b3e534180ca74bdd36bc546/ui-tests/src/test/java/org/openmrs/reference/page/RequestAppointmentPage.java#L35) for better java naming convention!